core/actions/data_preparation.ts

```diff
@@ -38,12 +38,23 @@ export class DataPreparation extends ActionBuilder<dataform.DataPreparation> {
 
     // Find targets
     const targets = getTargets(dataPreparationDefinition);
-    this.proto.targets = targets.map(target =>
-      this.applySessionToTarget(target, session.projectConfig, config.filename, true)
-    );
-    this.proto.canonicalTargets = targets.map(target =>
-      this.applySessionToTarget(target, session.canonicalProjectConfig)
-    );
+    this.proto.targets = targets
+        // Apply compilation overrides to target
+        .map(target =>
+            this.applySessionToTarget(target, session.projectConfig, config.filename, true)
+            // Apply data preparation defaults
```

Comment:
Nit: comment is just stating the name of the function, remove

Reply:
Done.